### DISCUSSION TITLE
Why did I only change the administration time, but PK changed?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1269] [Questions & Problems] [answered]
I'm working on a DDI project. I want to try what will happen if I give the perpetrator 1-4 hours in advance before giving the victim.
So I changed the administration time of the victim to 4h. However, as shown in the figure, I found that the PK curve of the victim and the perpetrator administered at the same time for 4H or 0h had some changes, and some parameters also had slight changes. Why does this happen?

Thank you in advance :)

![微信截图_20220801144539](https://user-images.githubusercontent.com/109507330/182090809-9d12122a-05b8-4244-a9d4-589e759786dc.jpg)

**Comment by AndreDlm** [comment, marked as answer]
Hi @noxavier, it could be because of different resolutions - make sure you have the same resolution for the two time intervals of your curves. You can find that here: 
![image](https://user-images.githubusercontent.com/26541603/182096964-423424b3-8fe9-4ece-96b3-5570fee3a434.png)
Maybe that helps

**Reply by noxavier** [reply]
That's it! Thank you very much!!
